src/main/java/com/imagelab/DashboardController.java

```diff
@@ -0,0 +1,51 @@
+package com.imagelab;
+
+import java.io.File;
+import java.io.IOException;
+import java.security.KeyStore;
+import java.util.List;
+
+import javafx.fxml.FXML;
+import javafx.scene.input.*;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.shape.Circle;
+
+public class DashboardController {
+
+    private final DataFormat circleFormat = new DataFormat("javafx.scene.shape.Circle");
+
+    @FXML
+    private AnchorPane buidlanchorpane;
```

Comment:
Naming should be consistent and Camel case. Please fix it in other relevant places as well.

Reply:
Sure, will check everwhere and fix this type of errors